tests/integration_tests/build/test_binary_szie.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Test that checks if the release binary size is within expected size"""
+
+import os
+import pytest
+
+import host_tools.cargo_build as host
+
+BINARY_SIZE_TARGET = 3137280
```

Comment:
The `firecracker` binary is where we're really interested, but please also add a check for the `jailer` binary size.

Reply:
Do you have the same binary size for both ARM and X86?

Reply:
Done